calico_kubernetes/tests/kube_plugin_test.py

```diff
@@ -1224,6 +1224,15 @@ def test_configure_logger(self, m_log_to_stdout, m_logging_f, m_logging_sh,
         else:
             m_log.addHandler.assert_called_once_with(f_handler)
 
+    def test_filter(self):
+        """Test filter method for IdentityFilterClass
+        """
+        record = Mock()
+        filter = logutils.IdentityFilter("ID")
+        returnVal = filter.filter(record)
+
+        self.assertEqual(record.identity, "ID")
```

Comment:
Use the `assert_true` and `assert_equals` functions for this to be more pythonic.